src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -156,6 +158,10 @@ def create_data_set(self, dataset_name, options = {}):
         json
         """
 
+        if options.get("like") is None:
+            if options.get("primary") is None and options.get("lrecl") is None:
```

Comment:
I believe this `and` should be an `or`, to throw an error when just one option is missing.

Reply:
Thank you for the feedback Timothy, I'll make the necessary changes!